src/test/java/fi/aalto/cs/apluscourses/presentation/CourseEndedBannerViewModelTest.java

```diff
@@ -7,14 +7,19 @@
 import fi.aalto.cs.apluscourses.intellij.model.CourseProject;
 import fi.aalto.cs.apluscourses.model.Authentication;
 import fi.aalto.cs.apluscourses.model.ModelExtensions;
+import fi.aalto.cs.apluscourses.model.User;
+import fi.aalto.cs.apluscourses.utils.observable.ObservableReadWriteProperty;
 import org.junit.Test;
 
 public class CourseEndedBannerViewModelTest {
   @Test
   public void testBanner() {
     var courseProject = mock(CourseProject.class);
```

Comment:
Use actual object instance instead of a mock here. Then call `courseProject.user.set(user)`. You can mock `Project` and `Authentication`. I think they can be just "empty" mock objects (no need to use `when` here anywhere).